presto-main/src/main/java/com/facebook/presto/server/QueryStateInfo.java

```diff
@@ -147,4 +152,10 @@ public String getUser()
     {
         return resourceGroupChain;
     }
+
+    @JsonProperty
+    public DateTime getCreateTime()
+    {
+        return createTime;
```

Comment:
An example JSON :  "2017-03-23T21:25:16.020Z"

It's a standard format that can be parsed into a DateTime in other languages as well. I think it's better than using unix timestamp which is not as readable. I also don't want to handle seconds vs milliseconds since epoch.